src/components/tab-bar/tab-bar.tsx

```diff
@@ -0,0 +1,139 @@
+import { Component, h, Prop, Element, State } from '@stencil/core';
+import {
+    MDCTabBar,
+    MDCTabBarActivatedEvent
+} from '@limetech/mdc-tab-bar';
+import { strings } from '@limetech/mdc-tab-bar/constants';
+import { Tab } from './tab.types';
+const { TAB_ACTIVATED_EVENT } = strings;
+
+@Component({
+    tag: 'limel-tab-bar',
+    styleUrl: 'tab-bar.scss',
+    shadow: true
+})
+export class TabBar {
+
+    @Element()
+    private element: HTMLElement;
+
+    /**
+     * List of tabs to display
+     */
+    @Prop()
+    public tabs: Tab[];
+
+    private mdcTabBar: MDCTabBar;
+
+    @State()
+    private selectedTab = this.tabs[0];
+
+    constructor() {
+        this.handleAction = this.handleAction.bind(this);
+
+    }
+
+    public componentDidLoad() {
+        this.element.focus();
+        this.setup();
+    }
+
+    private setup() {
+        this.element.focus();
```

Comment:
I don't think we should always set focus on the element when it's created. Instead, we should call `this.mdcTabBar.activateTab()` if one of the given tabs are active

Reply:
This code was for testing. The PR is pushed only for you guys to check if you guys workaround for shadow dom support